src/Essentials/src/SemanticScreenReader/SemanticScreenReader.android.cs

```diff
@@ -1,13 +1,18 @@
-﻿using Android.Views.Accessibility;
+﻿using System;
+using Android.Views.Accessibility;
 
 namespace Microsoft.Maui.Accessibility
 {
 	partial class SemanticScreenReaderImplementation : ISemanticScreenReader
 	{
 		public void Announce(string text)
 		{
-			AccessibilityManager manager = Android.App.Application.Context.GetSystemService(Android.Content.Context.AccessibilityService) as AccessibilityManager;
-			AccessibilityEvent announcement = AccessibilityEvent.Obtain();
+			var manager = Android.App.Application.Context.GetSystemService(Android.Content.Context.AccessibilityService) as AccessibilityManager;
+			var announcement = OperatingSystem.IsAndroidVersionAtLeast(33)
+				? new AccessibilityEvent()
```

Comment:
Technically the new ctor was available in API 30, but the `obtain` was deprecated in 33. I am going for existing behaviour all the way to 32 to not have any issues.